src/lib/runDev.js

```diff
@@ -44,6 +44,7 @@ const eventPoller = new EventPoller(fetchLogInterval)
 async function runDevLocal (config, cleanup, log) {
   const devConfig = cloneDeep(config)
   devConfig.envFile = path.join(config.app.dist, '.env.local')
+  const owJarFile = path.join(config.app.cliConfig.dataDir, 'openwhisk', 'openwhisk-standalone.jar')
```

Comment:
big +1 to reusing the same DIR across CLI updates
However, if I update a new version of the app plugin that relies on a new `openwhisk-standalone.jar` will it download the new jar, or still use the old one because it's already there?
If it still uses the old one I guess we need to version the file names or use a md5 to force the jar update

Reply:
This is true, the download location on disk is not determinant on the URL that might have 
changed. 

Two options:
1. We could take the last part of the URL path? `openwhisk/standalone-v1/openwhisk-standalone.jar`. If we stick to this format, deriving this will be easy.
2. We could hash the URL, and use it for the local path. e.g. https://bintray.com/api/ui/download/adobe/generic/openwhisk/standalone-v1/openwhisk-standalone.jar could hash to `acde0a`, thus the path can be `openwhisk/acde0a/openwhisk-standalone.jar`

Reply:
I like 1 ! it's readable in case the user wants to look at this.
Should we also clean up past versions (or open an issue and address this later if we get many more versions) ?

Reply:
I think we should revisit if it gets to be a problem. Perhaps a new command to list and clean (not sure).

Reply:
ok makes sense 🙂

Reply:
Latest commit should resolve the two comments above.